nebula/ui/components/VPNTipsAndTricksSection.qml

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Mozilla.VPN 1.0
+import Mozilla.VPN.qmlcomponents 1.0
+import components 0.1
+
+ColumnLayout {
+    id: layout
+
+    property string title
+    property string description
+
+    property string type
+    property var customFilter
+    property var featureTourCardVisible
```

Comment:
`featureTourCardVisible` either needs to bound to a default value or always be defined in your `ObjectModel` in `ViewTipsAndTricks`

also, lets change `var` to `bool`

Reply:
It is bound to a default value [here](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4319/files#diff-bfa251922231da58128bc4d90863ab50cae6fd6c0277d75b6159d70133b1d819R16) does it still need a default value in this view or is that one enough?

Reply:
I believe you are overwriting that binding when you [pass in your options here](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4319/files#diff-b037a6fe4a4973ce595c2734e31689591ae4d44b49f47b2f55e4fcefe220fee7R62)

Reply:
Hm, I thought QML would work like JavaScript. If the value is `undefined` it will go to the defaut. I'll fix it.

Reply:
I changed that line to cource an `undefined` into a `false`.